server/index.js

```diff
@@ -20,8 +20,8 @@ if (process.env.NODE_ENV === 'production') {
       process.exit(1);
     });
 } else {
-  db.sync({ force: true })
-    .then(seed)
+  db.sync()
```

Comment:
You aren't concerned the db might get messy?

Reply:
I think the better approach is to just manually run seed when you want to re-sync